src/bio/bio_monitor.c

```diff
@@ -623,6 +625,18 @@ bio_init_health_monitoring(struct bio_blobstore *bb, char *bdev_name)
 	D_ASSERT(channel != NULL);
 	bb->bb_dev_health.bdh_io_channel = channel;
 
+	/** register DAOS sensors to export NVMe stats */
+#define X(field, fname, desc, unit)					\
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon